src/lib/products.ts

```diff
@@ -82,6 +83,18 @@ export const products: { [product in Product]: ProductData } = {
 		github: "https://github.com/leaningtech/cheerpx-flash",
 		repositoryName: "cheerpx-flash",
 	},
+	"cheerpj-jnlprunner": {
+		id: "cheerpj-jnlprunner",
+		name: "CheerpJ JNLP Runner",
+		href: "/cheerpj-jnlprunner",
```

Comment:
I prefer `/cheerpj-jnlp-runner`

Reply:
Since I want to have an "extensions" section on the main page, we probably need to remove the jnlp-runner from product.ts. Should we have an extension.ts file instead?

Reply:
I have added the extensions on the product.ts, added the option "type", I think would be an idea to also move the "other tools" to product.ts, what do you think?

Reply:
Agree with moving "other tools" to products.ts. Possible issue is that the site will then expect docs content for them though.